qwandaq/src/main/java/life/genny/qwandaq/utils/MergeUtils.java

```diff
@@ -197,7 +196,7 @@ public Object wordMerge(String mergeText, Map<String, Object> entitymap) {
 					result = getFormattedString(sValue, matchFormat.group(1));
 					log.debug("String attribute " + attributeCode + " needs formatting. Format is " + entityArr[2] + ", Result is " + result);
 				} else {
-					result = beaUtils.getEntityAttribute(be.getRealm(), be.getCode(), attributeCode).getValueString();
+					result = be.getValueAsString(attributeCode);
```

Comment:
We should be fetching the entity attribute directly from the cache.

Reply:
beaUtils.getEntityAttribute(be.getRealm(), be.getCode(), attributeCode) always return null and no base entity found.
As base entity has attribute value itself, why shouldn't we use it ?